src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCard.swift

```diff
@@ -12,4 +12,6 @@ enum HomeStatisticsCard: Int32, CaseIterable {
 	case atLeastOneVaccinatedPerson = 5
 	case fullyVaccinatedPeople = 6
 	case appliedVaccinationsDoseRates = 7
+	case infectedPeopleHospitalizationRate = 8
+	case infectedPeopleInIntensiveCare = 9
```

Comment:
Wouldn't it make more sense to order this enum like the appearance of the tiles, so that `infectedPeopleHospitalizationRate` would have 3 and `infectedPeopleInIntensiveCare` 4. But only if this does not affect something other.

Reply:
No, the order comes from the background (current order is something like [2,1,8,9...]), it is aligned with the backend. If we change the enum, the data of the tiles will be switched too.

Reply:
okay, then it's fine for me